charts/prometheus-nginx-exporter/templates/deployment.yaml

```diff
@@ -4,10 +4,7 @@ kind: Deployment
 metadata:
   name: {{ include "prometheus-nginx-exporter.fullname" . }}
   namespace: {{ template "prometheus-nginx-exporter.namespace" . }}
-  {{- with .Values.additionalAnnotations }}
-  annotations:
-  {{ toYaml . | indent 4 }}
-  {{- end }}
+  annotations: {{ toYaml .Values.additionalAnnotations | nindent 4 }}
```

Comment:
There is nothing wrong with the previous code. In fact, we should prefer not to render an empty field. Applies to all the changes.

```yaml
{{- with .Values.additionalAnnotations }}
annotations: {{ toYaml . | nindent 4 }}
{{- end }}
```